ios/Podfile.lock

```diff
@@ -672,4 +672,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 41b806c7f131f87b716be1f1f9377532d6c9e43a
 
-COCOAPODS: 1.10.0
+COCOAPODS: 1.10.1
```

Comment:
These cocoapods changes should not be committed as I don't see any changes to pods.